{
    "title": "An issue was discovered in the Linux kernel through 4.17.10. There is an out-of-bounds access in __remove_dirty_segment() in fs/f2fs/segment.c when mounting an f2fs image.",
    "score": 5.5,
    "severity": "MEDIUM",
    "cwe_id": "CWE-476",
    "cvss_vector": "CVSS:3.0/AV:L/AC:L/PR:N/UI:R/S:U/C:N/I:N/A:H",
    "cpe_match": [
        "cpe:2.3:o:linux:linux_kernel:*:*:*:*:*:*:*:*"
    ],
    "cve_source": [
        "NVD"
    ],
    "ttpMapping": [
        {
            "techniqueId": "T1057",
            "confidence": "3.9",
            "techniqueName": "Process Discovery"
        },
        {
            "techniqueId": "T1059",
            "confidence": "3.0",
            "techniqueName": "Command and Scripting Interpreter"
        },
        {
            "techniqueId": "T1087",
            "confidence": "10.7",
            "techniqueName": "Account Discovery"
        },
        {
            "techniqueId": "T1213",
            "confidence": "3.6",
            "techniqueName": "Data from Information Repositories"
        },
        {
            "techniqueId": "T1548.003",
            "confidence": "2.0",
            "techniqueName": "Sudo and Sudo Caching"
        }
    ]
}
